ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -269,6 +295,13 @@ public BlobIdType getType() {
     return type;
   }
 
+  /**
+   * @return {@code true} if the blob that this id presents is encrypted. {@code false} otherwise
```

Comment:
represents or presents? Also, if it is false it could potentially still be encrypted, right?

Reply:
nope. if it is false, it can't be encrypted. Have fixed the docs.